src/Microsoft.AspNetCore.Razor.Language/RazorLanguageVersion.cs

```diff
@@ -1,33 +1,33 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System;
+
 namespace Microsoft.AspNetCore.Razor.Language
 {
-    public enum RazorLanguageVersion
+    public sealed class RazorLanguageVersion
     {
-        Version1_0 = 1,
+        public static readonly RazorLanguageVersion Version1_0 = new RazorLanguageVersion(1, 0);
```

Comment:
Suggestion: name these like `Version_1_0` instead. That reads a little nicer and it's what I did in MVC for this kind of thing.